src/OrchardCore.Modules/OrchardCore.Security/Options/ContentSecurityPolicyOptions/ReportUriContentSecurityPolicyOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace OrchardCore.Security.Options
+{
+    public class ReportUriContentSecurityPolicyOptions : ContentSecurityPolicyOptionsBase
+    {
+        public override string Name => ContentSecurityPolicyValue.ReportUri;
```

Comment:
This replaced by [report-to](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to) but still experimental